src/sage/sets/disjoint_set.pyx

```diff
@@ -834,6 +854,43 @@ cdef class DisjointSet_of_hashables(DisjointSet_class):
         cdef int j = <int> self._el_to_int[f]
         OP_join(self._nodes, i, j)
 
+    def make_set(self, new_elt = None):
```

Comment:
don't put spaces around `=` here. It should be `def make_set(self, new_elt=None):`